ignite/pkg/cosmosgen/templates/rest/rest.ts.tpl

```diff
@@ -1,45 +1,379 @@
-/* eslint-disable */
-import axios from 'axios'
-import * as qs from 'qs'
+// FOR REFERENCE ONLY -- NOT A TEMPLATE
```

Comment:
Could you please provide a more detailed description? What is a reference-only file? Also, wdyt add the line: 

// Code generated by Ignite <IGNITE VERSION>. DO NOT EDIT.

Reply:
Yep, merged! I'll open a follow up and attempt to generalize it as a go template + fill it in the HTTPRules data